spec/workers/commit_monitor_handlers/batch/github_pr_commenter/diff_content_checker_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe CommitMonitorHandlers::Batch::GithubPrCommenter::DiffContentChecker do
+  let(:batch_entry)        { BatchEntry.create!(:job => BatchJob.create!) }
+  let(:branch)             { create(:pr_branch) }
+  let(:diff)               { GitService::Diff.new(rugged_diff) }
+  let(:git_service_double) { double("GitService", :diff => diff) }
```

Comment:
Can you copy the pattern from my other PR for this? I thought it came out a little cleaner because 1) no need to call it a double in the let name and 2) You don't need the diff object anywhere in the spec, so might as well hide it right in the double creation.